### DISCUSSION TITLE
Link to UN metadata

### DISCUSSION THREAD

**Discussion by karihj** [open-sdg/open-sdg#1732] [Q&A]
Dear all, we can see that many sites using Open SDG have a field called "Link to UN metadata" under the Global metadata tab. 
The field links to a the metadata PDF provided for the indicator.
![image](https://user-images.githubusercontent.com/23165808/177869002-312a4d52-7a7e-4c6b-afe0-4885a784cecc.png)

As you can see this field is not visible on our site (see e.g. https://sdg.hagstova.fo/sdg-site/en/1-2-1/):

![image](https://user-images.githubusercontent.com/23165808/177869340-02ca863a-4434-4ead-9d3d-4a02b86bcd88.png)

Is there something that can be done to activate this field? 

Thanks in advance

**Comment by otis-bath** [comment]
Hi @karihj, thank you for raising this. Whilst this is not currently present in Open SDG it is something we are exploring and hoping to implement shortly as a site-wide configuration option. We will keep you updated - in the meantime if you wish to configure this for an individual indicator you can use this format in the indicator metadata:
`goal_meta_link: https://unstats.un.org/sdgs/metadata/?Text=&Goal=1&Target=1.1`
`goal_meta_link_text: United Nations Sustainable Development Goals Metadata (PDF 894 KB)`
